contrib/babelfishpg_common/src/collation.c

```diff
@@ -1674,3 +1705,11 @@ Oid bbf_get_like_collation(void)
 {
 	return like_cid;
 }
+
+void
+set_db_collation(Oid db_coll)
```

Comment:
Please add comment here that it expects valid db_coll and caller is responsible to check that. And also, report error if db_coll is invalid

Reply:
Updated

Reply:
report error if collation is invalid and also update comment.